packages/mui-material/src/Alert/Alert.d.ts

```diff
@@ -29,11 +58,43 @@ export interface AlertSlots {
 export type AlertSlotsAndSlotProps = CreateSlotsAndSlotProps<
   AlertSlots,
   {
+    /**
+     * Props forwarded to the root slot.
+     * By default, the avaible props are based on the [Paper](https://mui.com/material-ui/api/paper/#props) component.
+     */
+    root: SlotProps<React.ElementType<PaperProps>, AlertRootSlotPropsOverrides, AlertOwnerState>;
+    /**
+     * Props forwarded to the icon slot.
+     * By default, the avaible props are based on a div element.
+     */
+    icon: SlotProps<
+      React.ElementType<React.DetailsHTMLAttributes<HTMLDivElement>>,
+      AlertIconSlotPropsOverrides,
+      AlertOwnerState
+    >;
+    /**
+     * Props forwarded to the message slot.
+     * By default, the avaible props are based on a div element.
+     */
+    message: SlotProps<React.ElementType, AlertMessageSlotPropsOverrides, AlertOwnerState>;
```

Comment:
Can this be similar to the other two props?
```suggestion
    message: SlotProps<React.ElementType<React.DetailsHTMLAttributes<HTMLDivElement>>, AlertMessageSlotPropsOverrides, AlertOwnerState>;
```